packages/ag-charts-community/src/scene/util/fill.ts

```diff
@@ -1,12 +1,23 @@
 import { isArray, isObject } from 'ag-charts-core';
-import type { AgGradientColor } from 'ag-charts-types';
+import type { AgGradientColor, AgGradientColorBounds, AgGradientType, CssColor } from 'ag-charts-types';
 import type { AgPatternColor } from 'ag-charts-types';
 
-export function isGradientFill(fill: any): fill is AgGradientColor {
+export interface InternalAgGradientColor extends AgGradientColor {
```

Comment:
Could we extend this to allow `colorStops` to also be an array of strings, as that's what's generated by the themes?